rezasm-source/rezasm-core/src/simulation/transform/transformable.rs

```diff
@@ -27,9 +31,11 @@ impl Transformable {
             Transformable::FileReadTransformable(cursor) => {
                 Ok(RawData::from_int(cursor.clone(), simulator.get_word_size()))
             }
+            _ => Ok(RawData::new(&[])),
```

Comment:
no

Reply:
This fallthrough case is a bad idea. Any unhanded case should raise a compilation error.